volatility3/framework/plugins/windows/threads.py

```diff
@@ -18,6 +18,9 @@ class Threads(thrdscan.ThrdScan):
     _required_framework_version = (2, 4, 0)
     _version = (1, 0, 0)
 
+    def __init__(self):
+        self.implementation = self.list_process_threads
```

Comment:
missing call to super(), which should be before the self.implementation line so it doesn't get overwritten

Reply:
Good catch, thanks!